{"level":"error","message":"something failed during startupsomething failed during startup","stack":"Error: something failed during startup\n    at Object.<anonymous> (C:\\Node\\vidly-git\\index.js:34:7)\n    at Module._compile (internal/modules/cjs/loader.js:702:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:713:10)\n    at Module.load (internal/modules/cjs/loader.js:612:32)\n    at tryModuleLoad (internal/modules/cjs/loader.js:551:12)\n    at Function.Module._load (internal/modules/cjs/loader.js:543:3)\n    at Function.Module.runMain (internal/modules/cjs/loader.js:744:10)\n    at startup (internal/bootstrap/node.js:238:19)\n    at bootstrapNodeJSCore (internal/bootstrap/node.js:572:3)"}
{"level":"error","message":"rejected promiserejected promise","stack":"Error: rejected promise\n    at Object.<anonymous> (C:\\Node\\vidly-git\\index.js:39:26)\n    at Module._compile (internal/modules/cjs/loader.js:702:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:713:10)\n    at Module.load (internal/modules/cjs/loader.js:612:32)\n    at tryModuleLoad (internal/modules/cjs/loader.js:551:12)\n    at Function.Module._load (internal/modules/cjs/loader.js:543:3)\n    at Function.Module.runMain (internal/modules/cjs/loader.js:744:10)\n    at startup (internal/bootstrap/node.js:238:19)\n    at bootstrapNodeJSCore (internal/bootstrap/node.js:572:3)"}
